src/routes/v2/projects.rs

```diff
@@ -2558,3 +2561,134 @@ pub async fn delete_from_index(
+    markdown_options.extension.strikethrough = true;
+    markdown_options.extension.tagfilter = true;
+    markdown_options.extension.table = true;
+    markdown_options.extension.autolink = true;
+    markdown_options.extension.tasklist = true;
+
+    let string = info.into_inner().0;
+    let Some(project_data) = db_models::Project::get(&string, &**pool, &redis).await? else {
+        return Ok(HttpResponse::NotFound().body(""));
+    };
+    if !is_authorized(&project_data.inner, &None, &pool).await? {
+        return Ok(HttpResponse::NotFound().body(""));
+    }
+    let mut members_data =
+        db_models::TeamMember::get_from_team_full(project_data.inner.team_id, &**pool, &redis)
+            .await?;
+    let mut member_user_ids = members_data.iter().map(|x| x.user_id).collect::<Vec<_>>();
+    if let Some(oid) = project_data.inner.organization_id {
+        let organization_data = db_models::Organization::get_id(oid, &**pool, &redis).await?;
+        if let Some(organization_data) = organization_data {
+            let org_team = db_models::TeamMember::get_from_team_full(
+                organization_data.team_id,
+                &**pool,
+                &redis,
+            )
+            .await?;
+            for member in org_team {
+                if !member_user_ids.contains(&member.user_id) {
+                    member_user_ids.push(member.user_id);
+                    members_data.push(member);
+                }
+            }
+        }
+    }
+    let users = db_models::User::get_many_ids(&member_user_ids, &**pool, &redis).await?;
+    let versions = db_models::Version::get_many(&project_data.versions, &**pool, &redis).await?;
+    let mut versions = filter_authorized_versions(versions, &None, &pool).await?;
+    versions.sort_by(|a, b| b.date_published.cmp(&a.date_published));
+    let authors = users
+        .into_iter()
+        .map(|u| atom_syndication::Person {
+            uri: Some(format!("{site_url}/user/{}", &u.username)),
+            name: u.username,
+            email: None,
+        })
+        .collect::<Vec<_>>();
+
+    let project = models::projects::Project::from(project_data);
+    let mut feed = atom_syndication::Feed::default();
+    feed.title = atom_syndication::Text::plain(project.title.to_string());
+    feed.links.push(atom_syndication::Link {
+        href: format!("{site_url}/mod/{string}"),
+        rel: "alternate".to_string(),
+        mime_type: Some("text/html".to_string()),
+        title: Some(project.title.to_string()),
+        hreflang: None,
+        length: None,
+    });
+    feed.links.push(atom_syndication::Link {
+        href: format!("{self_addr}/project/{string}/feed"),
+        rel: "self".to_string(),
+        mime_type: Some("application/atom+xml".to_string()),
+        title: Some(project.title.to_string()),
+        hreflang: None,
+        length: None,
+    });
+    feed.icon = project.icon_url;
+    feed.id = format!("{site_url}/mod/{string}");
```

Comment:
Shouldn't this always use the project ID?
The Atom RFC [requires it to be *permanent and universally unique*](https://datatracker.ietf.org/doc/html/rfc4287#section-4.2.6), which would not be correct if the project's slug changes.

Reply:
Fixed